tools/bin/commands/install

```diff
@@ -53,7 +67,16 @@ install::run() {
     # Setup oc
     setup_oc
 
+    # Check for the proper setup
+    set +e
+    oc get syndesis >/dev/null 2>&1
```

Comment:
We loose output of the command here, I wonder if we could have it captured and displayed also to help troubleshooting